tendrl/commons/jobs/__init__.py

```diff
@@ -219,6 +230,16 @@ def process_job(job):
                              }
                 )
             )
+            alert_utils.alert_job_status(
+                "finished",
+                "Job finished successfully (job_id: %s)" % job.job_id,
+                integration_id=job.payload['parameters'].get(
+                    'TendrlContext.integration_id'
```

Comment:
Raise alert only when  `job.payload['parent'] is None` to ensure that this is not a child job.

Reply:
Done